app/views/course/condition/achievements/_form.html.slim

```diff
@@ -1,4 +1,8 @@
+- achievements = current_course.achievements
+- existing_conditions = @conditional.specific_conditions - [@achievement_condition]
+- available_achievements = achievements - existing_conditions.map(&:dependent_object)
```

Comment:
@jeremyyap There is still a possibility that when the `@conditional` is an `achievement`, it will be shown here as well? I think we should remove it too (even though the model validations take care of this on the backend).

Reply:
👌  will change in another PR